mint.json

```diff
@@ -724,9 +724,6 @@
       "inputPrefix": "ApiKey"
     }
   },
-  "integrations": {
-    "intercom": "fqe0apnx"
-  },
```

Comment:
What does it do? If it's only reporting events to intercom for visited pages, I would live it to help us target specific users later